airflow/contrib/hooks/gcp_compute_hook.py

```diff
@@ -16,14 +16,30 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-"""This module is deprecated. Please use `airflow.gcp.hooks.compute`."""
+"""This module is deprecated. Please use `airflow.gcp.hooks.compute.ComputeEngineHook`."""
```

Comment:
I think information here shouldn't contain specific hook name, just path to module (as it was before).

Reply:
Agree.

Reply:
Done